app/services/streaming/streaming.ts

```diff
@@ -614,7 +678,25 @@ export class StreamingService
 
     this.powerSaveId = remote.powerSaveBlocker.start('prevent-display-sleep');
 
-    obs.NodeObs.OBS_service_startStreaming();
+    if (this.views.isGreen) {
+      if (this.views.enabledPlatforms.length > 1) {
+        this.views.contextsToStream.forEach(async (contextName: string) => {
+          const context = this.videoSettingsService.contexts[contextName];
+          obs.NodeObs.OBS_service_setVideoInfo(context, contextName);
+          obs.NodeObs.OBS_service_startStreaming(contextName);
+
+          await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
what's the purpose of this?

Reply:
This is not necessary, just a failsafe to throttle the calls and prevent possible errors.